src/pages/_app.tsx

```diff
@@ -1,16 +1,40 @@
 // import '@/styles/globals.css'
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import { DSProvider } from "@nypl/design-system-react-components";
+import {
+  DSProvider,
+  useFeedbackBox,
+} from "@nypl/design-system-react-components";
+import React from "react";
+
+const onSubmit = async (values) => {
+  const response = await fetch("/api/feedback", {
+    method: "POST",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(values),
+  });
+};
 
 export default function App({ Component, pageProps }: AppProps) {
+  const { onOpen, isOpen, onClose, FeedbackBox } = useFeedbackBox();
```

Comment:
You can use the `FeedbackBox` component directly from the DS instead of the `useFeedbackBox` hook. The `onOpen`, `isOpen`, and `onClose` functions are not being used in a custom way in DC so it's okay to use the component directly. If you think that at some point we'll have a link that opens the form, then this should be kept. 

Here's an example: https://www.nypl.org/research/research-catalog/bib/b10833141
Click on the "contact a librarian" links and those will open the form. If DC will want/need similar functionality, then keep the `useFeedbackBox` hook.